widget/ui/src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -5,6 +5,7 @@ import { Typography } from '../Typography';
 
 export const TooltipContent = styled(RadixTooltip.Content, {
   zIndex: '999999',
+  fontFamily: '$widget',
```

Comment:
If you correctly pass the container to the tooltip container, it will inherit the font-family from the swap-box, allowing you to remove it from this location.

Reply:
fixed.